WalletWasabi.Fluent/Models/Transactions/PrivacySuggestionsModel.cs

```diff
@@ -386,25 +390,31 @@ await foreach (IEnumerable<SmartCoin> selection in selectionsTask.ConfigureAwait
 		return true;
 	}
 
-	private decimal GetDifferenceFiat(TransactionInfo transactionInfo, BuildTransactionResult transaction, decimal usdExchangeRate)
+	private (decimal BtcDifference, decimal FiatDifference) GetDifference(TransactionInfo transactionInfo, BuildTransactionResult transaction, decimal usdExchangeRate)
 	{
 		var originalAmount = transactionInfo.Amount.ToDecimal(MoneyUnit.BTC);
 		var totalAmount = transaction.CalculateDestinationAmount(transactionInfo.Destination);
 		var total = totalAmount.ToDecimal(MoneyUnit.BTC);
+		var btcDifference = total - originalAmount;
 		var fiatTotal = total * usdExchangeRate;
 		var fiatOriginal = originalAmount * usdExchangeRate;
 		var fiatDifference = fiatTotal - fiatOriginal;
 
-		return fiatDifference;
+		return (btcDifference, fiatDifference);
 	}
 
-	private string GetDifferenceFiatText(decimal fiatDifference)
+	private string GetDifferenceText(decimal btcDifference)
```

Comment:
How does it look like when it is "the same amount"?

Reply:
fixed https://github.com/zkSNACKs/WalletWasabi/pull/11463/commits/cf80e000e7b88b37c74269e0b153c9ff21f2b365